packages/backend/src/graphql/resolvers/wallet_address.test.ts

```diff
@@ -134,75 +133,68 @@ describe('Wallet Address Resolvers', (): void => {
       }
     )
 
-    test.each`
-      publicName
-      ${'Bob'}
-      ${undefined}
-    `(
-      'Can create a wallet address with additional properties (publicName: $publicName)',
-      async ({ publicName }): Promise<void> => {
-        input.publicName = publicName
-        input.additionalProperties = [
-          { key: '', value: '', visibleInOpenPayments: false },
-          { key: 'key', value: '', visibleInOpenPayments: false },
-          { key: '', value: 'val', visibleInOpenPayments: false },
-          { key: 'key', value: 'val', visibleInOpenPayments: false },
-          { key: 'key-public', value: 'val', visibleInOpenPayments: true }
-        ]
-        const response = await appContainer.apolloClient
-          .mutate({
-            mutation: gql`
-              mutation CreateWalletAddress($input: CreateWalletAddressInput!) {
-                createWalletAddress(input: $input) {
-                  code
-                  success
-                  message
-                  walletAddress {
-                    id
-                    asset {
-                      code
-                      scale
-                    }
-                    url
-                    publicName
+    test('Can create a wallet address with additional properties', async (): Promise<void> => {
+      input.publicName = 'Bob'
+      input.additionalProperties = [
+        { key: '', value: '', visibleInOpenPayments: false },
+        { key: 'key', value: '', visibleInOpenPayments: false },
+        { key: '', value: 'val', visibleInOpenPayments: false },
+        { key: 'key', value: 'val', visibleInOpenPayments: false },
```

Comment:
I just realized that I haven't thought of duplicating key's for additional properties. 
Do we want to:
1. generate an error and reject the create.
2. Only use the value for the last matching keys.

Reply:
That is an interesting case.

I actually think we should let them do it because it doesn't break nor complicate anything on our end and I dont want to assume there is no valid use-case for duplicate keys. Perhaps they want to group by key or something where only some of the values are private to open payments: 

```ts
// graphql update wallet address input
const input = {
  additionalProperties: [
    { key: 'myUserArray', value: 'publicUser1', visibleInOpenPayments: 'true' },
    { key: 'myUserArray', value: 'privateUser2', visibleInOpenPayments: 'false' },
    { key: 'myUserArray', value: 'publicUser2', visibleInOpenPayments: 'true' },
    { key: 'nickname', value: 'someNickname', visibleInOpenPayments: 'true' }
  ]
 }
 
 ...

// /GET wallet-address response
const response = {
  additionalProperties: [
    { key: 'myUserArray', value: 'publicUser1', visibleInOpenPayments: 'true' },
    { key: 'myUserArray', value: 'publicUser2', visibleInOpenPayments: 'true' },
    { key: 'nickname', value: 'someNickname', visibleInOpenPayments: 'true' }
  ]
}

// group values by 'myUserArray'
const userArrayValues = response.additionalProperties
  .filter(obj => obj.key === 'myUserArray')
  .map(obj => obj.value);

// users is ["publicUser1", "publicUser2"]
```

This is a capability we would remove by handling duplicate keys. Do they really want to do this? I have no idea - but  I think that's their concern and we shouldn't reduce their power to choose to use it this way, or other un-anticipated ways when it doesn't break nor complicate anything on our end.